scrapers/berlinstartupjobs.py

```diff
@@ -21,7 +21,7 @@ def scrape_bsj(category, old_time):
 
     results = {}
 
-    for i in range(len(post_elems)):
+    for i, __ in enumerate(link_elems):
```

Comment:
So I tried to understand what enumerate() does and how it differs from the for i in range. From what I understand, enumerate() gives an index for every incremental. So I could e.g. decide to skip a certain incremental based on the index. Now I'm not sure that I would need this in this use case. 

Is there another reason to choose enumerate() over for i in range?

I was thinking maybe it runs faster. So I ran the script with and without this change and keeping other local factors equal. The runtime was 11s for the version with for i in range and 13s for the version with enumerate().

Reply:
Ok. Did not time it. Thought also to be faster, since it calls an integrated Python iterator. As far as I heard it`s more pythonic and readable. But maybe in this context your solution (or another) is preferable. Other thought of mine was nesting all in a dictionary before and then work with Counter dict etc..

Reply:
No, took a look at it again. Think dict is it not worth in this case.

Reply:
I mean the 2s difference in run time might be influenced by outside factors, e.g. spotty wifi, higher ping in that moment, etc, so I wouldn't make a decision based on that.

Anyway, I was thinking about implementing code that I understand completely and can actually explain why it is used.

So regarding the pull request, I would delete this file from the request and merge the changes in the main module to the master branch.

Reply:
And maybe change to enumerate() at some point once I understand it a bit better :)

Reply:
Ah, I think I fucked this up by deleting the file. I think merging this pull request will then delete the file from the master branch. I can't find an option to undo that. I will add your changes to the main module separately and close this pull request, sorry about that.